lib/tasks/fix_cached_counters.rake

```diff
@@ -1,10 +1,15 @@
 desc 'Fix cached counters by reseting them to the correct count, in case they got corrupted somehow'
 task :fix_cached_counters => :environment do
   puts "Correcting cached counters: (this may take a few minutes)"
-  [ Architecture, Environment, Operatingsystem, Domain, Realm].each do |cl|
+  [ Architecture, Environment, Operatingsystem, Realm].each do |cl|
     cl.unscoped.all.each{|el| cl.reset_counters(el.id, :hosts, :hostgroups)}
     puts "#{cl} corrected"
   end
+  Domain.all.each do |el|
+    Domain.reset_counters(el.id, :hostgroups)
```

Comment:
Oh nevermind, I get it now.  I didn't even know this had a hostgroups count.  Why!?